pages/1.11/gui/services/index.md

```diff
@@ -23,3 +23,22 @@ Click the service name to open the Instances panel, which provides CPU, memory,
 For services with a web interface, hover over the service name and click ![open service](/1.11/img/open-service.png) to view it.
 
 **Tip:** You can access the Mesos web interface at `<hostname>/mesos`.
+
+# Service Status
+
+DC/OS UI introduces the following concepts to illustrate the possible status of a Service at any point in time:
+
+## Running
+The Service is reported as being successfully running.
```

Comment:
I'd remove "successfully" as the service is "running" if there is at least one "running" tasks and none of the other statuses match.

Reply:
Ok. Took this straight from the old Marathon docs. This hasn't changed right, you're just suggesting we improve it? Double checking. https://mesosphere.github.io/marathon/docs/marathon-ui.html#application-status-reference

Reply:
I believe that's what @orlandohohmeier is suggesting @leemunroe 

I think we have two options here:
1 - Keep what Lee took from Marathon documentation which is correct but vague if anyone wants to understand more.
2 - Keep the same plus add some clarification on how it works.

Running status only happens when there are no instances deploying nor recovering nor stopped in a service (this is not a suggested copy but only an explanation how it works)

Reply:
`The Service is reported as running and no instances are reported as deploying or recovering.` Better?